module/applications/actor/npc-sheet-2.mjs

```diff
@@ -119,7 +119,8 @@ export default class ActorSheet5eNPC2 extends ActorSheetV2Mixin(ActorSheet5eNPC)
     context.skills = Object.fromEntries(Object.entries(context.skills).filter(([, v]) => v.value));
 
     // Senses
-    context.senses.passivePerception = {
+    if (Object.hasOwn(this.actor.system.skills, 'prc')) {
```

Comment:
Got a syntax error here. As well as linting but it's probably enough to just check `if ( this.actor.system.skills.prc )`

Reply:
Serves me right me for not actually building it out. I'll sort in a sec.